dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/handler/NettyServerHandler.java

```diff
@@ -139,7 +138,7 @@ public void run() {
      * @throws Exception
      */
     @Override
-    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
why remove 'throws Exception'?

Reply:
Exception 'java.lang.Exception' is never thrown in the method

Reply:
If it occurs, it should also be handed over to the upper caller.